app/assets/javascripts/split_tabs.js

```diff
@@ -0,0 +1,29 @@
+$(function() {
+  var splitTabsContainer = ".split-tabs";
+  var splitTab = ".split-tab";
+  var splitTabPanel = ".split-tab-panel";
+  var activeClass = "is-active";
+  var transitionTime = 300;
+
+  $(splitTab + ":first-child")
+    .addClass(activeClass)
+    .closest(splitTabsContainer)
+    .siblings(splitTabPanel + ":first-of-type")
+    .show();
+
+  $(splitTab).click(function() {
+    $(this)
+      .siblings(splitTab)
+      .removeClass(activeClass)
+      .closest(splitTabsContainer)
+      .siblings(splitTabPanel)
+      .hide();
+
+    $(this)
+      .addClass(activeClass)
+      .closest(splitTabsContainer)
+      .siblings(splitTabPanel)
+      .eq($(this).index(splitTab))
+      .fadeIn(transitionTime);
+  });
```

Comment:
Thanks @alexandra-aas, that's a great idea. I'm going to leave this a little brittle for now though for the sake of time. 

@derekprior please feel free to refactor in another PR. I was spending a little too much time on it this morning and figure you or @margaretoti can knock it out quicker than I.